content/en/data_streams/live_messages.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Live Messages
+---
+
+{{< callout url="#" btn_hidden="true" header="Join the Preview!">}}
+Live Messages is in Preview for Java-Kafka services using Protobuf and Avro. If you're interested in other languages and technologies, reach out to support@datadoghq.com. 
+{{< /callout >}}
+
+Live Messages enables you to view a live tail of messages that a specific service consumes or produces. Accessing the live messages and inspecting the contents can help uncover issues when troubleshooting on a specific service.
+
+{{< img src="data_streams/live-messages.png" alt="Data Streams Monitoring with an open side panel, showing a live tail of Live Messages." style="width:100%;" >}}
+
+### Setup
+
+1. Enable [dynamic instrumentation][1] on the services you want to use this feature on. 
```

Comment:
I suggest just using Dynamic Instrumentation throughout for the sake of consistency, but I'm leaving it lowercase because it seems intentional, and may also be the style (since I'm also seeing "dynamic instrumentation" treated as a common noun in some cases on the Dynamic Instrumentation page).

Reply:
hmmmm yeah let's go with Dynamic Instrumentation